opal/mca/btl/usnic/btl_usnic_mca.c

```diff
@@ -189,7 +189,7 @@ int opal_btl_usnic_component_register(void)
 
     CHECK(reg_string("if_exclude",
                      "Comma-delimited list of usNIC devices/networks to be excluded (empty value means to not exclude any usNICs).  Mutually exclusive with btl_usnic_if_include.",
-                     NULL, &mca_btl_usnic_component.if_exclude,
+                     "lo", &mca_btl_usnic_component.if_exclude,
```

Comment:
Might want to put a comment here about why "lo" is a default value here (I know it might disappear in the next commit, but it is weird when you look at it -- so a short comment explaining it might be good).